car-hire-availability.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<!--Just playing around with some HTML on the new car hire optimisations, as practice. Would appreciate some feedback on how I've done here!-->
+
+<head>
+<title>Car Hire | Holiday rentals from £0.70 per day</title>
+</head>
+
+<body>
+<!--Not sure where the background image is added to, but I'm sure I'll work it out!-->
+<!--Is the nav bar actually a span rather than a div? One to double check-->
+  <div class="nav">
+    <div>Logo</div>
+    <div>Contact Us</div>
+  </div>
+
+  <div class="itinerary"></div>
```

Comment:
Quite a few divs from here onwards that seem to be empty – perhaps closed by accident? Did you mean to do this…
```
<div class="itinerary">
  <div class="itinerary_header">
    <div>Your Car Hire</div>
    <div>Need a hand? Call us on 0800 804 7362</div>
  </div>
  <div class="itinerary_content">
    <div>Pick up:</div>
    <div>Drop off:</div>
  </div>
</div>
```